xinference/model/llm/pytorch/core.py

```diff
@@ -73,23 +73,21 @@ def __init__(
         self._pytorch_model_config: PytorchModelConfig = self._sanitize_model_config(
             pytorch_model_config
         )
+        self._device = self._select_device()
```

Comment:
The model config will be displayed by the frontend. Do you think it is better to set the selected device back to the model config?